drivers/include/mma8x5x.h

```diff
@@ -40,9 +40,7 @@
 extern "C" {
 #endif
 
-#ifndef MMA8X5X_I2C_ADDRESS
```

Comment:
I see that the [datasheet](https://www.nxp.com/docs/en/data-sheet/MMA8453Q.pdf) of the MMA8453 shows that the address can be selected using a pin.

Reply:
Yes. You are right. the 84 series does come with an address pin.

Reply:
Removed the commit. Added to Compile Config in PR #13784